config/prow/cluster/cherrypicker_deployment.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: default
+  name: cherrypicker
+  labels:
+    app: cherrypicker
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: cherrypicker
+  template:
+    metadata:
+      labels:
+        app: cherrypicker
+    spec:
+      serviceAccountName: cherrypicker
+      terminationGracePeriodSeconds: 180
+      containers:
+      - name: cherrypicker
+        image: gcr.io/k8s-prow/cherrypicker:v20210303-204466f47d
+        imagePullPolicy: Always
+        args:
+        - --dry-run=false
+        ports:
+          - name: http
+            containerPort: 8888
+        volumeMounts:
+        - name: hmac
+          mountPath: /etc/webhook
+          readOnly: true
+        - name: oauth
+          mountPath: /etc/github
+          readOnly: true
+        - name: tmp
+          mountPath: /tmp
+      volumes:
+      - name: tmp
+        emptyDir: {}
+      - name: hmac
+        secret:
+          secretName: hmac-token
+      - name: oauth
+        secret:
+          secretName: oauth-token
```

Comment:
Is this the right token?

Reply:
I think so: https://github.com/kubernetes/test-infra/blob/2ee80aecbafb008d28756c0399b582ac20dc2775/config/prow/cluster/needs-rebase_deployment.yaml#L59-L61

Reply:
I mean, is k8s-ci-robot the right account for us to be doing this under

Reply:
True, good point. Is there one that can be used?

Reply:
I asked about this in the latest test-infra and @spiffxp mentioned that https://github.com/k8s-bot can be used. But I don't see any reference to a token of that bot in the repo, am I missing it?  @chaodaiG can you check if you see sth on the cluster?

Reply:
There is only one `oauth-token` like secret in k8s prow service cluster, and I believe that belongs to `k8s-ci-robot`

Reply:
Looks like this is the right token then!

Reply:
No it is unfortunately not, we can not use the same account as Prow itself as Prow ignores events from itself.